### PR TITLE
Try test with emulator in travis

### PR DESCRIPTION
**Pull request by chbatey**
Trying out the emulator i've been using locally in travis

Refs #2

### CODE REVIEW

**Comment by chbatey** [comment]
@patriknw this did not work! couldn't get gcloud to work with an emulator but now done instance creation with the grpc API and setup a spanner test to create/delete databases for each test and works in travis so is ready for rewview

File: journal/src/main/resources/GoogleInternetAuthorityG3.crt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+BggrBgEFBQcwAYYZaHR0cDovL29jc3AucGtpLmdvb2cvZ3NyMjAyBgNVHR8EKzAp
+MCegJaAjhiFodHRwOi8vY3JsLnBraS5nb29nL2dzcjIvZ3NyMi5jcmwwPwYDVR0g
+BDgwNjA0BgZngQwBAgIwKjAoBggrBgEFBQcCARYcaHR0cHM6Ly9wa2kuZ29vZy9y
+ZXBvc2l0b3J5LzANBgkqhkiG9w0BAQsFAAOCAQEAGoA+Nnn78y6pRjd9XlQWNa7H
+TgiZ/r3RNGkmUmYHPQq6Scti9PEajvwRT2iWTHQr02fesqOqBY2ETUwgZQ+lltoN
+FvhsO9tvBCOIazpswWC9aJ9xju4tWDQH8NVU6YZZ/XteDSGU9YzJqPjY8q3MDxrz
+mqepBCf5o8mw/wJ4a2G6xzUr6Fb6T8McDO22PLRL6u3M4Tzs3A2M1j6bykJYi8wW
+IRdAvKLWZu/axBVbzYmqmwkm5zLSDW5nIAJbELCQCZwMH56t2Dvqofxs6BBcCFIZ
+USpxu6x6td0V7SvJCCosirSmIatj/9dSSVDQibet8q/7UK4v4ZUN80atnZz1yg==
+-----END CERTIFICATE-----
```
Comment (johanandren) [inline journal/src/main/resources/GoogleInternetAuthorityG3.crt:25]: What is this cert? Some google self signed specific stuff?

File: journal/src/test/resources/logback-test.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <appender name="CONSOLE"
+  class="ch.qos.logback.core.ConsoleAppender">
+    <target>System.out</target>
+    <encoder>
+      <pattern>%date{MM/dd HH:mm:ss} %-5level[%thread] %logger{1} [%X{akkaSource}] - %m%n%xException</pattern>
```
Comment (johanandren) [inline journal/src/test/resources/logback-test.xml:7]: Huh, never saw the `%xException` before

File: journal/src/test/scala/akka/persistence/spanner/ExampleSpec.scala [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package akka.persistence.spanner
+
+class ExampleSpec extends SpannerSpec {
+  "Spanner db creation" should {
+    "work" in {}
+  }
+}
```
Comment (johanandren) [inline journal/src/test/scala/akka/persistence/spanner/ExampleSpec.scala:7]: Best example spec ever

File: journal/src/main/resources/GoogleInternetAuthorityG3.crt (hunk 1 above)
Comment (chbatey) [reply journal/src/main/resources/GoogleInternetAuthorityG3.crt:25]: Yeah i got it by hitting the google APIs. We could also require that is be installed into the local JVMs trust store

File: journal/src/test/scala/akka/persistence/spanner/ExampleSpec.scala (hunk 3 above)
Comment (chbatey) [reply journal/src/test/scala/akka/persistence/spanner/ExampleSpec.scala:7]: it connects and created a database, this made me very happy as it took some time to get it to (see commit history)

Comment (johanandren) [reply journal/src/test/scala/akka/persistence/spanner/ExampleSpec.scala:7]: What, you thought I was sarcastic?!

**Pull request merged**